funcao(dicionario[nome])

# e aqui depois da valor a isso sendo que na chamada da fun√ßao ele nao existe, mas eu quero fazer isso sem dar erro